src/main/java/com/elixr/poc/Constant/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.elixr.poc.Constant;
+
+/* this constant file holds all the error messages for all the end points */
+
+public class Constants {
+    public static final String DELETED = "Deleted";
+    public static final String ID_MISMATCH = "The ID is not Matching";
+    public static int REQUEST_VALUE;
```

Comment:
Dont put non final variable in Constant file

Reply:
Updated.